### PR TITLE
Release connectors with namespace change.

### PR DESCRIPTION
**Pull request by davinchia**
## What
Release all connectors affected by namespace change. Includes all JDBC sources and destinations.

Also add documentation for normalisation. Prerequisite to actually releasing.

## How
Bump all related connectors to `0.3.0`

## Pre-merge Checklist
- [x] *Run integration tests*
- [ ] *Publish Docker images*

## Recommended reading order
1. `namespaces.md`
1. various docker files and the source/destination definition list.

### CODE REVIEW

**Comment by jrhizor** [comment]
This needs to pick up the changes in https://github.com/airbytehq/airbyte/pull/2961 for `destination-postgres`. If namespacing isn't in the release we still need to do this before release.

**Comment by davinchia** [comment]
@jrhizor yeap. it will be in the release. I'll remove the acceptance test block after we release the related `postgres` connectors.

File: docs/architecture/namespaces.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Technical systems often group their underlying data into namespaces with each namespace's data isolated from another namespace. This isolation allow
+for better organisation and flexibility, leading to better usability.
+
+An example of namespace is a RDMS's `schema` concept. Some common use cases for schemas are enforcing permissiong, segregating test and production data
+and general data organisation.
+
+## Syncing
+The Airbyte protocol supports namespaces and allows Sources to define namespaces, and Destinations to write to a various namespaces.
+
+A sync conducted with a Source and Destination that both support namespaces will have its data replicated into the source-defined namespace. For such syncs,
```
[the remark this answers is not in the document]
Comment (cgardens) [reply docs/architecture/namespaces.md:13]: I would suggest you just add examples for each of the cases you mention. 

Cases:
source and destination support namespace
destination supports namespace but source does not
source supports namespace destination does not
nothing support namespace

just lay out the 4 cases and an example for each. you can maybe skip doing an example for the last case. 😂

**Review by cgardens — APPROVED** [review]

**Pull request merged**